app/views/users/two_factor_authentication_setup/index.html.slim

```diff
@@ -26,6 +25,7 @@ p.mt-tiny.mb0
           = radio_button_tag 'two_factor_setup_form[otp_method]', :voice
           span.indicator
           = t('devise.two_factor_authentication.otp_method.voice')
+  p = t('devise.two_factor_authentication.otp_method.instruction')
```

Comment:
@hursey013 updated at bae9c1c

looks very similar, but I hear ya on the semantics. Thanks for the feedback!

![screen shot 2017-03-29 at 5 28 15 pm](https://cloud.githubusercontent.com/assets/601515/24515724/51841e36-152d-11e7-9209-d81a4c94ab66.png)